packages/core/test/request.test.js

```diff
@@ -33,26 +34,33 @@ const httpHandler = (req, res) => {
   }
 };
 
-describe("redirect", () => {
+describe("request", () => {
+  const httpServer = http.createServer(httpHandler);
+  before(() => {
+    httpServer.listen(5136);
+  });
+
+  after(() => {
+    httpServer.close();
+  });
```

Comment:
I think the still need to be in the if to make the browser tests runnable

Reply:
Whoops that was a mistake. Fixed now. Thanks.